backend/bin/index.ts

```diff
@@ -0,0 +1,27 @@
+import app from '../src/app';
```

Comment:
index should be rather in 'src' than in 'bin'. This is a source file, not an .exe/.sh one